src/main/java/cbot/io/Parser.java

```diff
@@ -0,0 +1,362 @@
+package cbot.io;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+
+import cbot.task.Deadline;
+import cbot.task.Event;
+import cbot.task.Task;
+import cbot.task.TaskList;
+import cbot.time.TimeStuff;
+
+/**
+ * Handles much of the parsing of user inputs.
+ */
+public class Parser {
+    private Command c;
```

Comment:
Perhaps this variable can be renamed to be more descriptive? It is used throughout the entire class and it is not ideal to have to refer back constantly to see what `c` is.